core/src/services/ghac/docs.md

```diff
@@ -62,7 +62,7 @@ Refer to [`GhacBuilder`]'s public API docs for more information.
 
 ### Via Builder
 
-```no_run
+```rust
```

Comment:
Hello, we've intentionally used `no_run`. The code blocks are set to execute during documentation tests, but we want to prevent this because most services require configuration before running.

Reply:
Ah.... 😥